client/src/pages/trainer/courses/[id]/index.tsx

```diff
@@ -7,17 +6,18 @@ import Breadcrumbs from '@/src/shared/components/Breadcrumbs';
 import Tabs from '@/src/shared/components/Tabs';
 import Tab from '@/src/shared/components/Tabs/Tab';
 import Container from '@/src/shared/layouts/Container';
-import { useRouter } from 'next/router';
 import { Fragment, useEffect } from 'react';
 import { useAppDispatch } from '@/src/redux/hooks';
 import { reset } from '@/src/features/course/courseSlice';
+import type { GetServerSideProps, NextApiRequest, NextApiResponse } from 'next';
+import API from '@/src/apis';
+import type { AxiosError } from 'axios';
+import { getServerSession } from 'next-auth';
+import { settings } from '@/src/pages/api/auth/[...nextauth]';
 
-const CourseContent: React.FC = () => {
-  const { query } = useRouter();
+const CourseContent = ({ course }: { course: any }): JSX.Element | undefined => {
```

Comment:
instead of type `any`, can you give it a specific interface we have for course, please apply the same for learning path